src/bootstrap/flags.rs

```diff
@@ -525,18 +526,19 @@ Arguments:
                 Subcommand::Run { paths }
             }
             "setup" => {
-                let path = if paths.len() > 1 {
+                let profile = if paths.len() > 1 {
                     println!("\nat most one profile can be passed to setup\n");
                     usage(1, &opts, verbose, &subcommand_help)
                 } else if let Some(path) = paths.pop() {
-                    t!(path.into_os_string().into_string().map_err(|path| format!(
-                        "{} is not a valid UTF8 string",
-                        path.to_string_lossy()
-                    )))
+                    let profile_string = t!(path.into_os_string().into_string().map_err(
+                        |path| format!("{} is not a valid UTF8 string", path.to_string_lossy())
+                    ));
+
+                    profile_string.parse().expect("unknown profile")
```

Comment:
I think it would be nice to print the available profiles so you don't have to guess. Maybe add `Profile::all()` and loop over it?
```suggestion
                    profile_string.parse().unwrap_or_else(|_| {
                        eprintln!("error: unknown profile {}", profile_string);
                        eprintln!("help: the available profiles are:");
                        for choice in Profile::all() {
                            eprintln!("- {}", choice);
                        }
                        std::process::exit(1);
                    })
```

Reply:
thanks, done!